zokrates_core/src/proof_system/bn128/pghr13.rs

```diff
@@ -16,26 +20,51 @@ impl PGHR13 {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+struct PGHR13ProofPoints {
+    a: G1PairingPoint,
+    a_p: G1PairingPoint,
+    b: G2PairingPoint,
+    b_p: G1PairingPoint,
+    c: G1PairingPoint,
+    c_p: G1PairingPoint,
+    h: G1PairingPoint,
+    k: G1PairingPoint,
+}
+
+impl Proof<PGHR13ProofPoints> {
+    fn from_json(str: &str) -> Self {
+        serde_json::from_str(str).unwrap()
```

Comment:
same here

Reply:
Moved this method to https://github.com/Zokrates/ZoKrates/pull/583/files#diff-189d3ba6c93acd77029cea5c8a6a386aR26